### PR TITLE
Add settings option for weekly view day count visibility

### PR DESCRIPTION
**Pull request by KryptKode**
**Notes**
- add settings option for weekly view day count visibility
- should address this [feature request](https://github.com/SimpleMobileTools/Simple-Calendar/issues/1434)

**Screenshot**
Screenshot

### CODE REVIEW

File: app/src/main/kotlin/com/simplemobiletools/calendar/pro/fragments/WeekFragmentsHolder.kt [hunk 1]
```diff
@@ -49,7 +54,7 @@ class WeekFragmentsHolder : MyFragmentHolder(), WeekFragmentListener {
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
         weekHolder = inflater.inflate(R.layout.fragment_week_holder, container, false) as ViewGroup
         weekHolder!!.background = ColorDrawable(context!!.config.backgroundColor)
-
+        mConfig = requireContext().config
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/calendar/pro/fragments/WeekFragmentsHolder.kt:57]: do not store it here, just in onResume. Also rename "customise" to "customize"

**Comment by tibbi** [comment]
1. lets always show the week_view_days_count_divider or whatever is at the bottom, it is good to have something there
2. sometimes if I toggle the slider on and off, the slider itself is gone, even though the "7 days" label is visible

**Comment by KryptKode** [comment]
> 1. lets always show the week_view_days_count_divider or whatever is at the bottom, it is good to have something there
> 2. sometimes if I toggle the slider on and off, the slider itself is gone, even though the "7 days" label is visible

#2. Do you mean the toggle from the settings?

**Comment by tibbi** [comment]
well I mean using the new toggle you just added to flip the slider visibility

File: app/src/main/kotlin/com/simplemobiletools/calendar/pro/fragments/WeekFragmentsHolder.kt (hunk 1 above)
Comment (KryptKode) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/fragments/WeekFragmentsHolder.kt:57]: Do you mean to move initialisation to `onResume` ?

Comment (KryptKode) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/fragments/WeekFragmentsHolder.kt:57]: Like this
```
    override fun onResume() {
        super.onResume()
        val allowCustomizeDayCount = requireContext().config.allowCustomizeDayCount
        week_view_seekbar.beVisibleIf(allowCustomizeDayCount)
        week_view_days_count.beVisibleIf(allowCustomizeDayCount)
    }
```

Comment (tibbi) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/fragments/WeekFragmentsHolder.kt:57]: ye maybe, could be

Comment (tibbi) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/fragments/WeekFragmentsHolder.kt:57]: or actually handle null context too, anything can happen in onResume

Comment (KryptKode) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/fragments/WeekFragmentsHolder.kt:57]: Okay. Will do.

**Comment by tibbi** [comment]
I can still reproduce the slider missing by having the slider visibility turned off when starting the app with weekly view as default, going into the app settings, enabling slider and pressing Back. The "7 days" shows up, but the actual slider not

**Comment by KryptKode** [comment]
@tibbi  Please check.

**Pull request merged**